frontend/src/games/dot-count-game.tsx

```diff
@@ -0,0 +1,242 @@
+import { useState } from 'react';
+
+export type Dot = {
+  x: number;
+  y: number;
+};
+
+export type DotCanvasInfo = {
+  canvasWidth: number;
+  numberOfDots: number;
+  dots: Dot[];
+  dotRadius: number;
+};
+
+const noLastDotCountValue = 0;
+
+const defaultCanvasColor = 'aliceblue';
+const defaultCanvasDotColor = 'hotpink';
+
+const minDotCount = 1;
+const maxDotCount = 1000;
+const dotCountStep = 1;
+
+// In milliseconds
+const minDuration = 50;
+const maxDuration = 10000;
+const durationStep = 50;
+
+function delay(ms: number) {
```

Comment:
Ir jauciu sita funkcija reiktu iskelti i koki utils faila, nes visi ja naudojam:Dd

Reply:
Del sito sutinku, jeigu Aidas perdes visi pasinaudosim

Reply:
Jo, pakeiciau. I was thinking about that for a while as well.